### PR TITLE
update Intl.LocaleMatcher, eraDisplay to stage 1, Intl.Locale info to stage 2

### PR DESCRIPTION
**Pull request by longlho**

### CODE REVIEW

File: ecma402/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # [ECMAScript Internationalization API Specification](https://github.com/tc39/ecma402) proposals
 
- - [Stage 0 Proposals](stage-0-proposals.md)
- - [Finished Proposals](finished-proposals.md)
+- [Stage 0 Proposals](stage-0-proposals.md)
+- [Finished Proposals](finished-proposals.md)
```
Comment (ljharb) [inline ecma402/README.md:4]: i'll revert all these unrelated changes. please make sure your editor isn't blindly running prettier on all files - formatters should only run in projects that have them set up.

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by ljharb** [comment]
@longlho i'm confused; for localeMatcher to be at stage 1, a TC39 plenary must have occurred in which that gained consensus. 402 meetings do not have the capability of advancing stages. In which TC39 meeting did these advancements occur?

**Comment by longlho** [comment]
Oh sry I didn't know that. We can ignore this PR then.

**Comment by septs** [comment]
remove it?

**Comment by septs** [comment]
> remove it?

CC @ljharb

**Comment by ljharb** [comment]
PRs can’t ever be deleted, nor generally would be even if it were possible, so I’m not sure why you’re cc’ing me.

**Pull request closed without merging**